mvpa2/cmdline/cmd_ttest.py

```diff
@@ -58,14 +61,38 @@ def setup_parser(parser):
                         belong to.  E.g. if values are accuracies, it would be the
                         'greater', if errors -- the 'less'""")
 
+def guess_backend(fn):
+    if fn.endswith('.gz'):
```

Comment:
It might make it a bit shorter. Also probably a bit easier to understand.
There are now a few if-then-else statements that all deal with different backends. Why not use O-O to support the two backends?